components/subject/Summary.tsx

```diff
@@ -0,0 +1,283 @@
+        bgColor,
+      )}
+      title={title}
+      aria-label={title}
+      {...rest}
+    >
+      <Icon className="w-3 h-3" />
+      {count}
+      {text && <span>{text}</span>}
+    </button>
+  )
+}
+
+export const AccountStats = ({
+  stats,
+  showAll,
+  onAccountTakedownClick,
+}: {
+  showAll: boolean
+  onAccountTakedownClick?: () => void
+  stats: ToolsOzoneModerationDefs.AccountStats
+}) => {
+  const { takedownCount, suspendCount, appealCount, reportCount } = stats
+  // Dont want to make the UI noisy with appeal count if we're in summary mode and already showing takedown and suspension
+  const shouldShowAppeal =
+    appealCount && (showAll || !takedownCount || !suspendCount)
+  // Dont want to make the UI noisy with report count if we're in summary mode and we have both takedown and suspension count
+  const shouldShowReport =
+    reportCount && (showAll || (!takedownCount && !suspendCount))
+  const hasStats = takedownCount || suspendCount || appealCount || reportCount
+
+  return (
+    <>
+      {hasStats && <UserCircleIcon className="w-4 h-4 dark:text-gray-200" />}
+      {!!suspendCount && (
+        <StatView
+          appearance="danger"
+          count={suspendCount}
+          onClick={onAccountTakedownClick}
+          text={
+            showAll
+              ? pluralize(suspendCount, 'Suspension', { includeCount: false })
+              : ''
+          }
+          Icon={ShieldExclamationIcon}
+          title={`This account was suspended ${pluralize(
+            suspendCount,
+            'time',
+          )}`}
+        />
+      )}
+      {!!takedownCount && (
+        <StatView
+          appearance="danger"
+          count={takedownCount}
+          onClick={onAccountTakedownClick}
+          Icon={ShieldExclamationIcon}
+          text={
+            showAll
+              ? pluralize(takedownCount, 'Takedown', { includeCount: false })
+              : ''
+          }
+          title={`This account was taken down ${pluralize(
+            takedownCount,
+            'time',
+          )}`}
+        />
+      )}
+      {!!shouldShowAppeal && (
+        <StatView
+          appearance="warning"
+          count={appealCount}
+          onClick={onAccountTakedownClick}
+          text={
+            showAll
+              ? pluralize(appealCount, 'Appeal', { includeCount: false })
+              : ''
+          }
+          Icon={ExclamationTriangleIcon}
+          title={`This account appealed ${pluralize(appealCount, 'decision')}`}
+        />
+      )}
+      {!!shouldShowReport && (
+        <StatView
+          appearance="info"
+          count={reportCount}
+          Icon={FlagIcon}
+          text={
+            showAll
+              ? pluralize(reportCount, 'Report', { includeCount: false })
+              : ''
+          }
+          title={`This account was reported ${pluralize(reportCount, 'time')}`}
+        />
+      )}
+    </>
+  )
+}
+
+export const RecordsStats = ({
```

Comment:
Again, I suggest we
1) make a clear distinction between `0` and `undefined`, to make the business logic clearer in the code
2) that we do not use `undefined` as a reason not to show something (using a question mark in that case)